Userland/Services/Taskbar/main.cpp

```diff
@@ -72,6 +77,12 @@ int main(int argc, char** argv)
     auto window = TaskbarWindow::construct(move(menu));
     window->show();
 
+    // XXX: without the line comments at the end this line goes to 200 chars.
```

Comment:
If you want to avoid the long line, put the `|` characters at the start of each line instead of at the end :)

Reply:
Done. :^)